src/libraries/System.Private.CoreLib/src/System/Reflection/Emit/SymbolType.cs

```diff
@@ -440,18 +452,25 @@ public override Type GetNestedType(string name, BindingFlags bindingAttr)
             throw new NotSupportedException(SR.NotSupported_NonReflectedType);
         }
 
-        [DynamicallyAccessedMembers(GetAllMembers)]
+        [DynamicallyAccessedMembers(GetAllMembersInternal)]
         public override MemberInfo[] GetMember(string name, MemberTypes type, BindingFlags bindingAttr)
         {
             throw new NotSupportedException(SR.NotSupported_NonReflectedType);
         }
 
-        [DynamicallyAccessedMembers(GetAllMembers)]
+        [DynamicallyAccessedMembers(GetAllMembersInternal)]
```

Comment:
Is `GetAllMembers()` still used elsewhere?

Reply:
Same, exist in CoreLib and not accessible